src/logics/compound-v3/configs.ts

```diff
@@ -31,22 +32,16 @@ export const configs: Config[] = [
     markets: [
       {
         id: MarketId.USDC,
-        cometAddress: '0xc3d688B66703497DAA19211EEdff47f25384cdc3',
-        baseTokenAddress: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
+        comet: mainnetTokens.cUSDCv3,
+        baseToken: mainnetTokens.USDC,
       },
       {
         id: MarketId.ETH,
-        cometAddress: '0xA17581A9E3356d9A858b789D68B4d866e593aE94',
-        baseTokenAddress: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+        comet: mainnetTokens.cWETHv3,
+        baseToken: mainnetTokens.WETH,
```

Comment:
Actually, I was wondering if we could move common tokens like WETH to the common package to reduce the config size of protocols. What do you think?

Reply:
Agree the idea. We will build the common token list soon. After that, we can revisit this and update the tokens to use the common token list.